src/styles/_filtereditor.scss

```diff
@@ -67,17 +60,13 @@
   @extend .maputnik-icon-button !optional; /* stylelint-disable-line */
 }
 
-.maputnik-filter-editor-block-action {
-  margin-top: vars.$margin-2;
-  margin-bottom: vars.$margin-2;
-  display: inline-block;
-  width: 6%;
-  margin-right: 1.5%;
-}
+.maputnik-filter-editor-block {
+  display: flex;
+  margin-bottom: 2px;
 
-.maputnik-filter-editor-block-content {
-  display: inline-block;
-  width: 92.5%;
+  &-action {
```

Comment:
What will happen for RTL content for this rule?

Reply:
I reconsidered RTL content

![image](https://github.com/user-attachments/assets/d06de723-15d6-48d7-84ff-d26f2e0a60cc)
![image](https://github.com/user-attachments/assets/a470b769-4ed9-4cd3-8381-73de129c16aa)